src/lock/runner/runner-server.go

```diff
@@ -0,0 +1,75 @@
+package runner
```

Comment:
Thinking more about this package name, I think we want something more like `server` since the server runs runners. There isn't too much about the runners in this code